hotham/src/contexts/physics_context.rs

```diff
@@ -32,7 +33,12 @@ impl Default for PhysicsContext {
         let (contact_force_send, contact_force_recv) = crossbeam::channel::unbounded();
         let event_handler = ChannelEventCollector::new(collision_send, contact_force_send);
         let gravity: Matrix3x1<f32> = vector![0.0, 0.0, 0.0]; // TODO: no gravity in SPACE baby! But some games may uh, need this.
-        let integration_parameters = IntegrationParameters::default();
+        let integration_parameters = IntegrationParameters {
+            dt: 1. / 72.,
```

Comment:
This constant should probably be extracted from here.

Reply:
Good point! :D I have done that now in #372